### DISCUSSION TITLE
How to remove border ?

### DISCUSSION THREAD

**Discussion by patrickus33** [templaza/astroid-framework#483] [Q&A]
Hello,
How can I remove the border created by bootstrap on pages with only one article?
With 2 columns, no problem, I want to keep the border.
I can't find the option in the menu configuration.
You can see what I mean here.
https://snipboard.io/UxXq9e.jpg

Thank you !

**Comment by Chacapamac** [comment]
You have to add custom css.

This website is not an Astroid WebSite?

**Reply by Chacapamac** [reply]
@patrickus33 

> Yes, it's an atroid site

Ok I see, my bad.

**Try something like that.**

```
/*----- FOR BLOG LAYOUTS -----*/
/* Articles Surronding box design (featured) */
.article-wraper-inner {box-shadow:none !important;}
div.article-wraper-inner { background-color: transparent;}

```
——————**ENHANCEMENT** ✅ ——————
I also mark this  post as an enhancement because it is rare that modern websites use borders and drop-shadow in a clean design.
@sonvnn 
Maybe some controls directly in the “**Article/Blog**” area of the template administration on the border, corners and background surrounding Article views?
![Screen Shot 2023-12-02 at 12 26 04 AM](https://github.com/templaza/astroid-framework/assets/1907276/dad52bbf-7365-4af2-8710-a222d4a8b8f4)

**Reply by patrickus33** [reply]
You're right, it would be a great idea !
Would it be possible to offer a choice of bootstrap classes like "shadow" or "border" ?
